website/docs/r/app_service.html.markdown

```diff
@@ -219,7 +219,7 @@ Additional examples of how to run Containers via the `azurerm_app_service` resou
 
 * `php_version` - (Optional) The version of PHP to use in this App Service. Possible values are `5.5`, `5.6`, `7.0`, `7.1`, `7.2`, and `7.3`.
 
-* `python_version` - (Optional) The version of Python to use in this App Service. Possible values are `2.7` and `3.4`.
+* `python_version` - (Optional) The version of Python to use in this App Service. Possible values are `2.7`, `3.6`, `3.7` and `3.8`.
```

Comment:
(as above) 3.4 will still be supported for existing App Services - so can we make this:

```suggestion
* `python_version` - (Optional) The version of Python to use in this App Service. Possible values are `2.7`, `3.4`, `3.6`, `3.7` and `3.8`.
```